app/assets/stylesheets/characters.scss

```diff
@@ -40,8 +40,13 @@
   border: 1px $border_color_character_iconless solid;
   width: 102px;
   height: 102px;
+  padding: 20px 5px;
   text-align: center;
 
+  color: $font_color_page;
```

Comment:
Without this, on styles that have a specific color for link hover, we see the text inside the icon change color on hover, which seems strange (it seems like it should be "fixed" as a property of the image).